src/plugin/helpers/test.ts

```diff
@@ -81,6 +81,7 @@ export const createEditorWithElements = <
     nodeSpec,
     getNodeFromElementData,
     getElementDataFromNode,
+    elementDataValidator,
```

Comment:
One thought on naming – functions are verbs, not nouns – what do you think about `validateElementData`, which describes what this function does, rather than what it is?